### ISSUE TITLE
[Bug]: Dependency lines

### ISSUE THREAD

**Issue by MrHimera** [FTBTeam/FTB-Mods-Issues#1183]
### Mod

FTB Quests

### Mod version

ftb-quests-forge-2001.4.3

### Forge / Fabric version

Forge 1.20.1 47.2.23

### What issue are you having?

Previously, there was a dark line between quests with Linear Progression Mode, and a light line with Flexible Progression Mode. Now all the lines are light, is this a bug or is it supposed to be like this?

### Steps to reproduce

1. Create a couple of quests with Linear Progression Mode and link them
2. Look at the line of dependence between them

**Comment by MrHimera** [comment]
> I never made dependency lines a different colour between flexible and linear mode?

Here is an example from different versions. The top line is quests with Linear Progression Mode, and the bottom line is quests with Flexible Progression Mode.

FTB Quests 2001.4.3
![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/29342708/b9861672-15c3-4eea-a96c-913e8deea23e)

FTB Quests 2001.3.0
![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/29342708/82daed7c-8ddf-406c-b52c-b8257da9673c)

**Comment by desht** [comment]
OK, fair enough. I'll amend my previous statement: "I never intentionally made dependency lines a different colour between flexible and linear mode" :laughing: 

But I guess it's behaviour that could be restored. Not sure how important it is, but ok.

**Comment by MrHimera** [comment]
> OK, fair enough. I'll amend my previous statement: "I never intentionally made dependency lines a different colour between flexible and linear mode" 😆
> 
> But I guess it's behaviour that could be restored. Not sure how important it is, but ok.
> 
> Update: this commit changed the way dependency line colouring is handled: [FTBTeam/FTB-Quests@8537a76](https://github.com/FTBTeam/FTB-Quests/commit/8537a764d310d584b3a4e68d1f4bdf841b5ddb1a)
> 
> That commit allows theming of uncompleted quest dependency line colours in the `ftb_quest_theme.txt` resource file, and doesn't distiniguish linear vs. flexible quest modes. How important is that really, as far as the player is concerned? I could add some equivalent behaviour back, but I'm not sure I get the importance of dependency line colouring for linear vs. flexible...?

I have a lot of quests, both linear and flexible. And it was very convenient to distinguish them by eye, rather than poking and looking in the settings.

**Comment by desht** [comment]
Fair enough, I guess I can darken the dependency lines again if it's for a flexible-mode quest.
